<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Problem Reporting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="lie_subject"/>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="txe_content"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Object</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Message</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Thumbnail</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="Thumbnail" name="item_thumbnail">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>90</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Click to take a screenshot.</string>
       </property>
       <property name="accessibleName">
        <string>item thumbnail</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pub_send">
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Thumbnail</class>
   <extends>QLabel</extends>
   <header>..thumbnail</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
